src/index.ts

```diff
@@ -410,5 +416,10 @@ export default class Quote implements BlockTool {
    */
   _toggleTune(tune: Alignment) {
     this._data.alignment = tune;
+
+    // Dispatch change if quoteElement already exists
+    if (this._quoteElement) {
+      this.api.blocks.getBlockByElement(this._quoteElement)?.dispatchChange();
```

Comment:
you don't need to find a Block via API. It is passed to the tools contructor

https://editorjs.io/tools-api/#class-constructor

Reply:
Done 👍🏼